src/client/src/redux/reducers/questions-entities-reducer.js

```diff
@@ -0,0 +1,24 @@
+import actionTypes from "../actions/constants";
+
+const initialState = {};
+
+const questionsEntitiesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.FETCH_QUESTIONS_SUCCESS: {
+      const { questions } = action;
+      return { ...questions };
+    }
+
+    case actionTypes.REMOVE_QUESTION_REQUEST: {
+      const { question } = action;
+      const questionsEntities = { ...state };
+      delete questionsEntities[question.id];
```

Comment:
maybe we should mark the question as done instead of deleting but I agree that deleting is easier and even more readable